src/Symfony/Component/Mailer/Transport/SendmailTransport.php

```diff
@@ -99,6 +100,18 @@ protected function doSend(SentMessage $message): void
 
         $this->stream->setCommand($command);
         $this->stream->initialize();
+        if (false !== strpos($command, ' -t')) {
+            $email = $message->getOriginalMessage();
+            if ($email instanceof Email) {
+                foreach ($email->getBcc() as $recipient) {
+                    $this->stream->write('Bcc:'.$recipient->toString()."\n");
+                }
+            } else {
+                foreach ($message->getEnvelope()->getRecipients() as $recipient) {
+                    $this->stream->write('Bcc:'.$recipient->toString()."\n");
```

Comment:
I'm not sure this is a good idea.

Basically, if somebody send an email to `foo`, the payload sent to `sendmail` will contain `to: foo` **and** `Bcc: foo`...

Do you have a better idea to find the missing recipients?...

Reply:
the `Bcc` should always be in the `$message->getOriginalMessage()->message` so maybe walk through `$message->getOriginalMessage()->toIterable()` until you get the `Content-Type` header?

But feels hacky :grimacing: